README.md

```diff
@@ -20,10 +20,6 @@ if let Some((Width(w), Height(h))) = size {
 }
 ```
 
-## Minimum Rust Version
-
-This crate requires a minimum rust version of 1.63.0 (2022-08-11)
-
```

Comment:
I'd prefer to keep this info in the README, at least for now until more tooling has native support for the `rust-version` field (for example, I don't think crates.io exposes the `rust-version` field at the moment)

Reply:
OK, done.